docs/tutorial/installation.rst

```diff
@@ -38,6 +39,10 @@ http://localhost:15672/. The username is ``guest`` and the password is
 read the messages in the queues. Keep it open in a browser tab, we'll need it
 later.
 
+If your project uses Docker, a conatiner needs to be set up for RabbitMQ.
```

Comment:
Not everyone uses Docker to manage and run their containers, so I'd recommend phrasing this as:
"If your project uses containers, consult the `RabbitMQ documentation`_ on its container."